TrackletAlgorithm/MatchEngineUnit.h

```diff
@@ -209,15 +195,15 @@ inline void step(const VMStubMECM<VMSMEType> stubmem[4][1<<(kNbitsrzbinMP+kNbits
   inline void processPipeLine(ap_uint<1> *table) {
 #pragma HLS inline
 
-    auto stubindex=stubdata____.getIndex();
-    auto stubfinez=stubdata____.getFineZ();
-    auto stubfinephi=stubdata____.getFinePhi();
-    auto stubbend=stubdata____.getBend();
+    auto stubindex=stubdata___.getIndex();
+    auto stubfinez=stubdata___.getFineZ();
+    auto stubfinephi=stubdata___.getFinePhi();
+    auto stubbend=stubdata___.getBend();
     constexpr int absz = (1 << MatchEngineUnitBase<VMProjType>::kNBitsBuffer) - 1;
     const bool isPSStub = VMProjType==BARREL ? LAYER <= TF::L3 : 
-                                               LAYER <= TF::D2 ? ((zbin____ & absz) < 3) || ((zbin____ & absz) == 3 && stubfinez <= 3) :
-                                                                 ((zbin____ & absz) < 3) || ((zbin____ & absz) == 3 && stubfinez <= 2);
-    auto stubbendReduced=stubdata____.getBendPSDisk();
+                                               LAYER <= TF::D2 ? ((zbin___ & absz) < 3) || ((zbin___ & absz) == 3 && stubfinez <= 3) :
+                                                                 ((zbin___ & absz) < 3) || ((zbin___ & absz) == 3 && stubfinez <= 2);
```

Comment:
@aryd I realize you didn't introduce them here, but is there any way to get rid of the magic 2 and 3 here? I don't know if they can be derived from existing constants, but at least they can be assigned to meaningfully named constants either here or in Constants.h.

Reply:
I addressed this by adding a comment to explain this code; It was not clear to me that introducing three variables for these constants would really make the code more easy to read. However, I think we should be able to calculate these from other constants, in particular the radius of the innermost 2S module.